ui/src/views/Settings.vue

```diff
@@ -19,40 +19,187 @@
         />
       </cv-column>
     </cv-row>
-    <cv-row>
+    <cv-row v-if="error.configureModule">
+      <cv-column>
+        <NsInlineNotification
+          kind="error"
+          :title="$t('action.configure-module')"
+          :description="error.configureModule"
+          :showCloseButton="false"
+        />
+      </cv-column>
+    </cv-row>
+    <cv-row
+      v-if="
+        loading.getConfiguration ||
+        loading.getAvailableInterfacesBeforeConfiguration
+      "
+    >
       <cv-column>
         <cv-tile light>
-          <cv-form @submit.prevent="configureModule">
-            <!-- TODO remove test field and code configuration fields -->
-            <cv-text-input
-              :label="$t('settings.test_field')"
-              v-model="testField"
-              :placeholder="$t('settings.test_field')"
-              :disabled="loading.getConfiguration || loading.configureModule"
-              :invalid-message="error.testField"
-              ref="testField"
-            ></cv-text-input>
-            <cv-row v-if="error.configureModule">
-              <cv-column>
-                <NsInlineNotification
-                  kind="error"
-                  :title="$t('action.configure-module')"
-                  :description="error.configureModule"
-                  :showCloseButton="false"
-                />
-              </cv-column>
-            </cv-row>
-            <NsButton
-              kind="primary"
-              :icon="Save20"
-              :loading="loading.configureModule"
-              :disabled="loading.getConfiguration || loading.configureModule"
-              >{{ $t("settings.save") }}</NsButton
-            >
-          </cv-form>
+          <cv-skeleton-text width="100px" class="mg-bottom-lg" />
+          <cv-skeleton-text width="70%" heading class="mg-bottom-lg" />
```

Comment:
I checked the documentation of the component but i can't obtain the same effect with paragraph and linecount because of the different widths that i need. Maybe there is something that i don't know, but also the input skeleton doesn't work for me in this situation.